docs/api_reference.rst

```diff
@@ -9,13 +9,40 @@ methods in the current release of PyMC experimental.
   :maxdepth: 2
 
 
+:mod:`pymc_experimental`
+=============================
+
+.. automodule:: pymc_experimental.marginal_model
```

Comment:
Do you want all the docstrings for all the functions rendered in the same page? (like it currently happens) or listing the functions in the api page and then have individual pages for each docstring or submodule?

Reply:
Individual pages is probably better in the long term. Is it too much trouble if I ask you to push the needed changes?

Reply:
I can try to do it tomorrow